### DISCUSSION TITLE
Robotdyn Relay Configuration

### DISCUSSION THREAD

**Discussion by Snubbeke** [arendst/Tasmota#16329] [Q&A] [answered]
I managed to flash a Robotdyn Relay with an ESP8266 chip but I am stuck how to configure it as a module in Tasmota. There is close to nothing on Google and Robotdyn disappeared from the internet including support.
My guess was to use GPI04 (see attached picture) and assign relay in the module configuration but the relay does not click on the module when I toggle. Any advise?
![ESP8266R40A_char](https://user-images.githubusercontent.com/94734821/186443347-86802790-bff4-46fd-9c81-d66c0652c45c.jpg)
![ESP8266R40A_5](https://user-images.githubusercontent.com/94734821/186443360-853b7579-c3de-45aa-8697-7a8c04eefde7.jpg)
![ESP8266R40A_6](https://user-images.githubusercontent.com/94734821/186443364-2c210a98-7f88-46ee-a41f-c3aabf1077d3.jpg)

**Comment by sfromis** [comment]
Gpio 4 is not very likely for the relay, precisely because it is made available on those convenient pins. A common pin for relay is gpio 12, but you can try gpio 12-16, 5 or 2, more or less in that order.

**Reply by Snubbeke** [reply]
Ok, I will give it a try.

**Comment by Snubbeke** [comment, marked as answer]
It was 14. Thanks a lot!

**Reply by freedomlives** [reply]
How did you get it to program? I've tried flipping RX/TX, pressing reset button, etc, but nothing gets the flashing tool (tasmotizer nor esp_flasher) to recognize it.

**Reply by sfromis** [reply]
To flash ESP8266, you generally need to ground gpio0 before/while applying power (not using the reset button). My *guess* would be that the "00" in the above pinout may be gpio0.

**Reply by freedomlives** [reply]
Thanks so much! Long time since I've flashed over serial! I'd never used this board, so was thinking I just got a dud so many years ago!
BTW, the GPIO2, for anyone who missed the tiny 02 half under the connector.
